### ISSUE TITLE
Showcase Quarto Extensions for the Julia Community

### ISSUE THREAD

**Issue by pat-alt** [pat-alt/quarto-julia-website#5]
Add the presentation for [Quarto Extensions for the Julia Community](https://pretalx.com/juliacon2024/me/submissions/JGT893/). Just need to fill the [presentation.qmd](https://github.com/pat-alt/quarto-julia-website/blob/21de06c78a36d2bb89a8584c771721077b88c4f1/examples/contributed/juliacon_2024/quarto_extensions/presentation.qmd#L2) file.

cc @kellertuer

**Comment by kellertuer** [comment]
I started my (second) part of the talk in the `presentation.qmd`, but since I was not sure whether the main branch is directly rendered to your homepage, I pushed it to a branch, see https://github.com/pat-alt/quarto-julia-website/tree/kellertuer/Quarto-JC24-start/examples/contributed/juliacon_2024/quarto_extensions

but when coming here to comment I saw you also have a `3-...` branch for that. Should I merge that there?

My main problem (besides checking the content a bit) is currently that even with as many `eval: false` as I could spare, I have a Julia block within a markdown block that is still executed, same for my Julia block that is part of my original `make.jl`. These should not be run but I am not able to deactivate that (and that code blocks within code blocks are run is also very strange).

So for both – the branch to actually use and this evaluation issue – help wanted :)
